client/src/app/(modules)/datasets/page.tsx

```diff
@@ -124,14 +124,14 @@ export default function DatasetsModule() {
                     </Button>
                   </PopoverTrigger>
                   <PopoverContent
-                    className="w-[330px] overflow-y-auto rounded-none border border-gray-400 p-0 text-base shadow-none"
+                    className="w-[330px] overflow-y-auto rounded-none p-0 text-base shadow-md"
```

Comment:
You should bring back the old value of the class here. The styles are intentionally different (from the ones in the forms) so that all the filters have the same look: no shadow and a dark gray border when opened.

```suggestion
                    className="w-[330px] overflow-y-auto rounded-none border border-gray-400 p-0 text-base shadow-none"
```